docs/source/preparation_input_dataset.rst

```diff
@@ -12,6 +15,73 @@ For more details, please refer to
 `its documentation <http://docs.datalad.org/en/stable/generated/man/datalad-create.html>`_,
 and `DataLad Handbook <https://handbook.datalad.org/en/latest/basics/101-101-create.html>`__.
 
+==============================================================================
+Raw (unzipped) BIDS dataset, or zipped BIDS dataset?
+==============================================================================
+
+* If there are zip files, like ``sub-*.zip``,
+  it would be considered as a zipped BIDS derivatives dataset.
+* If there are only unzipped folders, like folder ``sub-*/``,
+  then it is considered as a raw (unzipped) BIDS dataset.
+* If both zipped file ``sub-*.zip`` and unzipped folders ``sub-*`` present,
```

Comment:
I think having both zipped and unzipped files in the SAME input dataset would cause problems. You can have multiple input datasets where one is zipped and one isn't, but within the same input dataset, the data need to be uniformly structured.